beetsplug/permissions.py

```diff
@@ -103,7 +103,9 @@ def set_permissions(self, files=[], dirs=[]):
                 'setting file permissions on {}',
                 util.displayable_path(path),
             )
-            os.chmod(util.syspath(path), file_perm)
+            perm = stat.S_IMODE(os.stat(util.syspath(path)).st_mode)
+            if perm != file_perm:
```

Comment:
We already have a helper function for this check above, I think it would be nice to re-use that here:

```suggestion
            if not check_permissions(syspath(path), file_perm)):
```

Probably, it would be cleaner to change that to use ` S_IMODE`?

Reply:
Feedback actions, please check the latest push. Thanks!